### ISSUE TITLE
cannot exclude files/folders in subdirectories in _site.yml

### ISSUE THREAD

**Issue by rubenarslan** [rstudio/rmarkdown#951]
To reproduce:

Put a file in a project subfolder, try to ignore it via `exclude: ['subfolder/bla.txt']`. 

When the site is knit, the file "bla.txt" is copied anyway.

It doesn't matter whether the thing in the subdirectory is a file or a folder. 
I also naively expected "subfolder/*" to work, but it didn't.

**Comment by friendly** [comment]
Is there any update on this?  `exclude: ["folder1/", "folder2/"]` seems to be rather basic, and should just work.

**Comment by yihui** [comment]
@friendly No, we haven't got a chance to look at this issue. If you use blogdown, you will be able to exclude any files/dirs.

**Comment by rubenarslan** [comment]
@friendly IIRC excluding entire subfolders works (but might have to omit the slash), this issue was about single files in subfolders.

**Comment by friendly** [comment]
Ah, omitting the slash **works** for folders `notes/` and `other-slides/` in the line below.

    exclude: ["notes", "other-slides", "*.pptx", "*.bak", "*.doc"]

Specific extensions, like `*.pptx` or `*.bak` are **not** excluded in subfolders

**Comment by NMarkgraf** [comment]
The OLD slash problem in rmarkdown when it comes to folders … What about handling all dirs with "/" as Long as possible and only directly before using it (in Windows enviroment and when it is needed (!!!)) converting it? ;-)

**Comment by clauswilke** [comment]
Just came across this issue. It also arises when using the distill package, as far as I can tell.
